archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII

```

Comment:
`pathNameAscii` is kind of redundant if you already have `pathNameUTF8`. The only stuff that `pathNameAscii` might provide is throwing exceptions on non-ASCII character codes (and that's usually not what you want).

Generally, I'm not sure that it would be super useful. There are tons of non-UTF8 encodings even  if we're talking only about Western countries and only about UNIX world: iso8859-1, iso8859-2, iso8859-7, iso8859-15, cp437, etc - see https://www.freebsd.org/doc/handbook/using-localization.html - and for end-user, doing something like `.toString("whatever")`, or, in worst case, `KaitaiStream.bytesToString(..., "whatever")`, is probably not that hard.